naucse/views.py

```diff
@@ -920,3 +936,75 @@ def course_calendar_ics(course):
             abort(404)
 
     return Response(str(calendar), mimetype="text/calendar")
+
+def active_runs():
+    today = datetime.date.today()
+
+    runs = (model.runs_from_year(today.year) +
+            model.runs_from_year(today.year - 1) +
+            model.runs_from_year(today.year - 2))
+    ongoing = [run for run in runs if
+                run.start_date <= today and run.end_date >= today]
+
+    return len(ongoing)
+
+def all_runs():
+    all_years = list(model.safe_run_years.keys())
+
+    return sum([len(model.runs_from_year(year)) for year in all_years])
+
+def sessions_delivered():
+    today = datetime.date.today()
+    all_years = list(model.safe_run_years.keys())
+
+    all_runs = []
+    for year in all_years:
+        all_runs.extend(model.runs_from_year(year))
+
+    all_sessions = []
+    for run in all_runs:
+        all_sessions.extend(list(run.sessions.values()))
+
+    return len([session for session in all_sessions if session.date < today])
+
+def sessions_available():
+    uniq_sessions = {}
+    for course in safe_courses():
```

Comment:
Tohle spíš na @hroncok nebo @encukou: má být v této statistice jen počet kanonických sessions nebo i sessions co byly jen na kurzech? A nebylo by možná indikativnější/hezčí mít počet materiálů než lekcí, případně také mít tento počet?